contents/blog/introduction-to-customer-retention.md

```diff
@@ -5,7 +5,7 @@ rootPage: /blog
 sidebar: Blog
 showTitle: true
 hideAnchor: true
-author: joe-martin
```

Comment:
I feel this is very much co-authored - I assume that would break things by adding 2 - happy either way who is the author.

Reply:
Yeah, I've had a quick look at trying to add two authors but it just gets a bit messy in terms of links, profile images, etc. We could fix it, but it doesn't feel like it's the biggest problem to work on at the moment. 

I'll put your name on it. There are definitely enough articles online with my name on them already! :D